openslides_backend/action/motion_submitter/create.py

```diff
@@ -36,4 +37,15 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             raise ActionException(
                 "Cannot create motion_submitter, meeting id of motion and (temporary) user don't match."
             )
+
+        # check, if (user_id, motion_id) already in the databse.
+        filter = And(
+            FilterOperator("user_id", "=", instance["user_id"]),
+            FilterOperator("motion_id", "=", instance["motion_id"]),
+        )
+        db_instances = self.database.filter(
+            collection=self.model.collection, filter=filter, mapped_fields=["user_id"],
+        )
+        if len(db_instances) > 0:
```

Comment:
you can uses the `exists(...)` function of the datastore for this

Reply:
Okay, updated.